extensions/extensions-configuration-store-vault/src/main/java/com/boozallen/aissemble/configuration/dao/VaultPropertyDao.java

```diff
@@ -50,14 +50,13 @@ public boolean checkEmpty() {
      * @param groupName to read from the vault
      * @param propertyName to read from the vault
      */
-    @Override
+    @Override 
     public Property read(String groupName, String propertyName) {
         try {
             final Vault vault = getVault();
             // read from Vault
             final LogicalResponse readResponse = vault.logical()
                     .read(String.format("aissemble-properties/%s/%s", groupName, propertyName));
-
```

Comment:
Would clean up this file to remove the whitespace changes

Reply:
cleaned up the file